pages/projects/projects-spacetimecube.tsx

```diff
@@ -102,6 +121,46 @@ const ProjectSpaceTimeCube: NextPage<Props> = ({
       });
     },
   );
+  const options = {
+    responsive: true,
+    plugins: {
+      legend: {
+        position: "top" as const,
+      },
+      title: {
+        display: true,
+        text: "projects in selected year",
+      },
+    },
+  };
+
+  const labels = Array.from(
+    selectedYear &&
+      projectsByYearCountry.filter((d) => d.year == selectedYear)[0]?.countries
+      ? projectsByYearCountry
+          .filter((d) => d.year == selectedYear)[0]
+          .countries.keys()
+      : [],
+  );
+
+  const data = {
+    labels,
+    datasets: [
+      {
+        label: selectedYear,
+        data: Array.from(
+          selectedYear &&
+            projectsByYearCountry.filter((d) => d.year == selectedYear)[0]
+              ?.countries
+            ? projectsByYearCountry
+                .filter((d) => d.year == selectedYear)[0]
+                .countries.values()
+            : [],
+        ),
+        backgroundColor: "rgba(53, 162, 235, 0.5)",
```

Comment:
Maybe we can use one of the brand colors `itc_green` itc_blue` here?

Reply:
How to add itc_green and itc_blue, using Clsx?

Reply:
Hm… I think you cant use the `className` prop (neither with nor without `clsx()`) here – most probably you can only assign a color string to the `backgroundColor` property. But you can use the tailwind config something like this:

```js
import resolveConfig from "tailwindcss/resolveConfig";
import tailwindConfig from "/tailwind.config.js";

…

const twConfig = resolveConfig(tailwindConfig);
const itc_green = (twConfig.theme.colors["itc-green"].DEFAULT);

… 

const data ={ …, backgroundColor: itc_green, …}
```